### ISSUE TITLE
LSTM Support for the TFLM

### ISSUE THREAD

**Issue by 1msec** [tensorflow/tflite-micro#1243]
Is there any plan to support lstm for the TFLM?

**Comment by giovanni6inve** [comment]
Add   AddUnidirectionalSequenceLSTM(); in  all_ops_resolver.cpp file and after it works:)

**Comment by 1msec** [comment]
Thank you for your reply!
Let me ask more about it.

I made my own model using LSTM cell.
And I converted it into a tflite model using the convert guide.
I got the below after converting.
I found the there were no LSTM(or RNN) layers in the model.

Here are my questions.
1. Which layer or operation is matched to the "UnidirectionalSequenceLSTM" you mentioned.
   (Let me know so I can change my Keras Model, I'd like to use LSTM)
2. Could you teach me which layer or operation is matched to LSTM Cell that I used in my Keras model.

> **My Model** 
    my_model = tf.keras.Sequential()
    cell = tf.keras.layers.LSTMCell(units=num_classes, input_shape=(sequence_length, input_dim))
    my_model.add(tf.keras.layers.RNN(cell=cell, return_sequences=True))
    my_model.add(tf.keras.layers.TimeDistributed(tf.keras.layers.Dense(units=num_classes, activation='softmax')))
    my_model.compile(loss='categorical_crossentropy', 
                    optimizer=tf.keras.optimizers.Adam(learning_rate=learning_rate),
                    metrics=['accuracy'])

> **tflite converted** 
    <img width="274" alt="image" src="https://user-images.githubusercontent.com/106607582/179457098-cb4a83d8-db90-4a32-a425-53ff845440df.png">

Thank you in advance :)

**Comment by christophsiegl** [comment]
> Add AddUnidirectionalSequenceLSTM(); in all_ops_resolver.cpp file and after it works:)

@giovanni6inve  I tried out this and it works (at least on linux_x86_64), but it takes quiet a long time to run! But why is this opcode not added per default in all_ops_resolver.cc? This makes no sense to me? 

>     1. Which layer or operation is matched to the "UnidirectionalSequenceLSTM" you mentioned.
>        (Let me know so I can change my Keras Model, I'd like to use LSTM)

@1msec  Try to use keras.layers.LSTM()! This should work.
